recipes/amqp-cpp/all/conanfile.py

```diff
@@ -45,7 +45,7 @@ def config_options(self):
 
     def requirements(self):
         if self.options.get_safe("linux_tcp_module"):
-            self.requires("openssl/1.1.1d")
+            self.requires("openssl/1.1.1g")
```

Comment:
It's hard to keep up with openssl 😄 
```suggestion
            self.requires("openssl/1.1.1h")
```